app/models/imported_item.rb

```diff
@@ -17,25 +17,25 @@ def import(params)
       contacts_rows << row
     end
 
-    unique_contacts, not_unique_contacts = unique_records contacts_rows
+    unique_contacts, not_unique_contacts = unique_and_non_unique_records contacts_rows
 
     contacts = []
-    unique_contacts.each do |row|
+    unique_contacts.each do |contact_record|
       contacts << Contact.new(
-        test_and_trace_account_id: row[0],
-        nhs_number: row[1],
-        is_vulnerable: (row[2] == 1 || row[2].to_s.downcase == 'true'),
-        first_name: row[3],
-        surname: row[4],
-        date_of_birth: row[5],
-        email: row[6],
-        mobile: row[7],
-        telephone: row[8],
-        address: row[9],
-        postcode: row[10],
-        needs: [Need.new(category: 'Check in', start_on: Date.today + 2.days, name: row[11])],
-        test_trace_creation_date: row[12],
-        isolation_start_date: row[13],
+        test_and_trace_account_id: contact_record[0],
```

Comment:
not 100% sure contact_record is clearer than row - row makes it clear it is coming from spreadsheet - but not a big deal I don't think